public/app/plugins/datasource/loki/modifyQuery.ts

```diff
@@ -393,6 +393,16 @@ function addLabelFormat(
   return newQuery;
 }
 
+// Add filter as line filter
```

Comment:
```suggestion
// Add filter as line filter
```
I think it's redundant.